### PR TITLE
Add datachannel support for relay message transport.

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

**Comment by JonathanLennox** [comment]
See also https://github.com/jitsi/jicofo/pull/1051

File: jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt [hunk 1]
```diff
@@ -373,7 +373,21 @@ class Colibri2ConferenceHandler(
         if (c2relay.create) {
             val transBuilder = Transport.getBuilder()
             transBuilder.setIceUdpExtension(relay.describeTransport())
-            respBuilder.setTransport(transBuilder.build())
+            c2relay.transport?.sctp?.let {
```
Comment (bgrozev) [inline jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt:368]: Should we throw if SCTP is disabled in the config?

File: jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt [hunk 2]
```diff
@@ -76,13 +75,6 @@ class Colibri2ConferenceHandler(
             if (!RelayConfig.config.enabled) {
                 throw IqProcessingException(Condition.feature_not_implemented, "Octo is disable in configuration.")
             }
-            if (!WebsocketServiceConfig.config.enabled) {
```
Comment (bgrozev) [inline jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt:79]: Should we still throw if neither SCTP nor websockets are enabled?

File: jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt (hunk 1 above)
Comment (JonathanLennox) [reply jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt:368]: It already does, above.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**